.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
 ## What? and Why?
-> What does this pull request change/fix? Why was it nessessary?
+> What does this pull request change/fix? Why was it necessary?
```

Comment:
Is this wrong in the actual template in other repos? If so then worth raising tech debt to fix it